webapp/src/pages/createForm/components/Section.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import Question from './Question';
+
+const Section = ({
+  t, num, setSection, sections, inputs,
+  addSection, addQuestion
+}) => {
+  const [input, setInput] = useState({
+    name: inputs.name,
+    description: inputs.description,
+    id: inputs.id,
+    questions: inputs.questions
+  });
+
+  const handleChange = (prop) => (e) => {
+    setInput({...input, [prop]: e.target.value});
+  }
+  const updateSections = () => { // Update sections in the application form when section loses focus
+    sections[num-1] = input;
+    setSection([...sections]);
+  }
+  const handleQuestions = (inpt) => {
+    setInput({...input, questions: inpt})
+  }
+  return (
+    <>
+      <div
+        className="section-wrapper"
+        // onBlur={updateSections}
+      >
+        <div className="section-number">
+          <label>{t('Section') + ` ${num} ` + t('of') + ` ${sections.length}`  }</label>
+        </div>
+        <div className="title-description">
+          <div className="section-header">
+            <input
+              type="text"
+              value={input.name}
+              onChange={handleChange('name')}
+              className="section-inputs section-title"
+              onBlur={updateSections}
+            />
+            <button
+              className="title-desc-toggle"
+              name="title-desc-toggle"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i className='fa fa-ellipsis-v fa-lg fa-dropdown'></i>
+            </button>
+            <div className="dropdown-menu" aria-labelledby="title-desc-toggle">
+              <button
+                className="delete-section"
+                onClick={() => addSection()}
+              >
+                {t("Add Section")}
```

Comment:
I think it's a bit unintuitive that the "Add Section" action is inside a Section's menu... It probably makes more sense to have add section be a button on the Form.